vscode/src/formView.ts

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import { begin, end } from './html';
+import { formStyles } from './styles';
+import data from './templateDetails.json';
+
+export function getFormView(templateSelected: string): string {
+
+    let templateObject = data.find(x => x.id == templateSelected);
+    let templateId = templateSelected;
+    let templateName = templateObject.name;
+    let templatePlaceholders = templateObject.placeholders;
+
+    let beginForm: string = `
+            <h2 style="margin-left:20px;">${templateName}</h2>
+            <div class="container">
+                <form action="" id="myForm" onsubmit="myFunction()">`;
+
+    let i = 0;
+    let elements: string = "";
+    templatePlaceholders.forEach(element => {
+        elements += `
+        ${element.id}: elements[${i}].value,`;
+        i++;
+    });
+
+    let scriptHandling = `
+            <script>
+                function myFunction(template) {
+                    var elements = document.getElementById("myForm").elements;
```

Comment:
Unused local variable "elements"

Reply:
This variable is used from within elements string created in the TypeScript code.